ci/docker/install/ubuntu_scala.sh

```diff
@@ -22,10 +22,17 @@
 
 set -ex
 # install libraries for mxnet's scala package on ubuntu
+echo 'Installing Scala...'
 apt-get install -y software-properties-common
 add-apt-repository -y ppa:webupd8team/java
 apt-get update
 echo "oracle-java8-installer shared/accepted-oracle-license-v1-1 select true" | debconf-set-selections
 apt-get install -y oracle-java8-installer
 apt-get install -y oracle-java8-set-default
-apt-get update && apt-get install -y maven
\ No newline at end of file
+
+echo "deb https://dl.bintray.com/sbt/debian /" | tee -a /etc/apt/sources.list.d/sbt.list
+apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2EE0EA64E40A89B84B2DF73499E82A75642AC823
+apt-get update && apt-get install -y \
+    maven \
+    sbt \
+    scala
```

Comment:
Could we pin the scala version?

Reply:
I'd like to pin everything, not just scala, but let's put that out of scope for now.